src/components/Rubric.js

```diff
@@ -1,177 +1,171 @@
-          </label>
-        </div>
-      </td>
-      <td>{props.text}</td>
-    </tr>
-  )
-}
-
-class Rubric extends Component {
-
-  constructor(props) {
-    super(props);
-    this.state = {
-      activeCompetencyIndex: 0
+        </a>
+      </li>
+    )
+  })
+
+  class Rubric extends Component {
+
+    constructor(props) {
+      super(props);
+      this.state = {
+        activeCompetencyIndex: 0
+      }
+      this.icons = [ "dashboard", "explore", "code", "backup", "lock", "bug_report", "line_style", "perm_identity", "star_rate" ]
+      this.renderCompetencies = this.renderCompetencies.bind(this);
+      this.renderCompetencyButtons = this.renderCompetencyButtons.bind(this);
+      this.renderCriteriaForLevel = this.renderCriteriaForLevel.bind(this);
+      this.renderLevels = this.renderLevels.bind(this);
+      this.getIsFetching = this.getIsFetching.bind(this);
     }
-    this.icons = [ "dashboard", "explore", "code", "backup", "lock", "bug_report", "line_style", "perm_identity", "star_rate" ]
-    this.renderCompetencies = this.renderCompetencies.bind(this);
-    this.renderCompetencyButtons = this.renderCompetencyButtons.bind(this);
-    this.renderCriteriaForLevel = this.renderCriteriaForLevel.bind(this);
-    this.renderLevels = this.renderLevels.bind(this);
-    this.getIsFetching = this.getIsFetching.bind(this);
-  }
 
-  componentWillMount() {
-    const id = this.props.match.params.id;
-    this.props.getAssessment(localStorage.getItem("userId"), id);
-  }
-
-  setActiveComp(index) {
-    this.setState({
-      activeCompetencyIndex: index
-    })
-  }
+    componentWillMount() {
+      const id = this.props.match.params.id;
+      this.props.getAssessment(localStorage.getItem("userId"), id);
+    }
 
-  renderCompetencyButtons() {
-    if (this.props.assessment.rubricJSON) {
-      return this.props.assessment.rubricJSON.Competencies.map((comp, index) => {
-        var active = "";
-        if (index === this.state.activeCompetencyIndex) {
-          active = "active";
-        }
-        return (
-          this.getIsFetching() ? "" :
-            <CompetencyButton
-              name={comp.name}
+    renderCriteriaForLevel(level, compIndex) {
+      if (this.props.assessment.assessmentObject.rubricJSON.Competencies) {
+        return this.props.assessment.assessmentObject.rubricJSON.Competencies[compIndex].Scales.map((scale, index) => {
+          // if the criteria level matches the level parameter, add the
+          // criteria component
+          return scale.Criteria.filter(criteria => criteria.level === level).map((criteria, index) => {
+            return <Criteria
               key={uuidv1()}
-              index={index}
-              isActive={active}
-              icon={this.icons[index]}
-              setActiveComp={this.setActiveComp.bind(this)}
-            />
-        )
-      })
+              answer={criteria.answer}
```

Comment:
you can deconstruct here too:

`const { answer, id, text } = criteria;` => then you don't have to type criteria so often.